core/templates/dev/head/pages/terms/terms.html

```diff
@@ -39,7 +39,7 @@ <h1>Terms of Use</h1>
         GUARDIAN, YOU MUST NOT REGISTER FOR OR ACCESS THIS WEBSITE.
       </p>
 
-      <a class="oppia-about-anchor" name="our-services"></a>
+      <a class="oppia-privacy-terms" name="our-services"></a>
```

Comment:
Ditto -- oppia-privacy-terms-anchor, here and below

Reply:
Done.